contrib/fuzz/oss_fuzz_build.sh

```diff
@@ -38,6 +38,9 @@ compile_fuzzers() {
     done
 }
 
+cd $SRC/instrumentation
+go run main.go $SRC/containerd/images
```

Comment:
Yes, "instrumentation" is being cloned here: https://github.com/google/oss-fuzz/pull/8504

Reply:
Thanks! Can you mention the location in oss_fuzz_build.sh? It may be obvious for oss-fuzz folks, but not so for containerd folks.

Compared to cncf-fuzzing, https://github.com/AdamKorcz/instrumentation doesn't have much containerd stuff. So I'm fine having a separate repos for that.

Reply:
Also I'd like to merge this PR after https://github.com/google/oss-fuzz/pull/8504. Would it work for you?

Reply:
> Thanks! Can you mention the location in oss_fuzz_build.sh? It may be obvious for oss-fuzz folks, but not so for containerd folks.

In an inline comment or in this PR?

> Compared to cncf-fuzzing, https://github.com/AdamKorcz/instrumentation doesn't have much containerd stuff. So I'm fine having a separate repos for that.

Yes, the instrumentation project is meant to be general.

> Also I'd like to merge this PR after https://github.com/google/oss-fuzz/pull/8504. Would it work for you?

Sure, it has just been merged.

Reply:
> In an inline comment or in this PR?

In the shell script itself.

Reply:
> In the shell script itself.

Not sure I understand. Should I write in `oss_fuzz_build.sh` where it is in the containerd repo? If users have already found the `oss_fuzz_build.sh` script, do they need to know where it is?

Reply:
oh sorry, I meant, oss_fuzz_build.sh should explain how does it get main.go. You could just say something like "this is from https://github.com/AdamKorcz/instrumentation"

Reply:
Thanks for the elaboration and sorry for the misunderstanding. A comment has been added!